quickevent/app/quickevent/plugins/Event/src/services/emmaclient.cpp

```diff
@@ -443,14 +443,18 @@ void EmmaClient::exportStartListRacomTxt()
 		lastId = id;
 		int si = q2.value("runs.siId").toInt();
 		int startTime = q2.value("runs.startTimeMs").toInt();
+		bool startTimeCardNull = q2.value("runs.startTimeMs").isNull();
```

Comment:
`card_start_time_is_null` use `snake_case` for local variables, pls see https://github.com/Quick-Event/quickbox/wiki/Code-style

Reply:
whole code committed by you :)

Reply:
Ok, I'll try to fix it in function / class.